\documentclass[notitlepage]{article}
\usepackage{color,soul,amsmath,graphicx}
\usepackage[outercaption]{sidecap}
\usepackage[citestyle=numeric,backend=bibtex]{biblatex}
\usepackage[font=small,labelfont=bf]{caption}
\usepackage{verbatim}
\sidecaptionvpos{figure}{c}
\providecommand{\abs}[1]{\lvert#1\rvert}
\providecommand{\norm}[1]{\lVert#1\rVert}
\bibliography{library}

\title{On the quantitative inter-dependence between gene regulation, protein levels and growth rate}
\begin{comment}
\author{Leeat Keren, Uri Barenholz, Ron Milo}
\end{comment}
\date{April 2014}

\begin{document}
\maketitle
\abstract{
  In many microorganisms, the proteome composition changes dramatically as a function of the growth environment.
Furthermore, many of these changes seem to be coordinated with the growth rate.
However, although cellular growth rates, gene expression levels and gene regulation have been at the center of biological research for decades, their quantitative interdependence remains largely unexplained.

We analyzed global trends in two recently published proteome composition data sets for the model microorganism \emph{E.coli} under various growth conditions, and specifically, their relation to growth rate.
We found that the cellular concentration of a large fraction of the proteins measured coordinately increased with the growth rate.
This fraction includes proteins spanning different functional groups and proteins that are involved in different cellular processes.
We developed a simple model that explains how such a widely coordinated increase in the concentration of many proteins can be the result of passive redistribution of resources, due to active down regulation of a few unrelated proteins.
Our model further explains why and how such changes relate to the growth rate under different environmental conditions.

We conclude that, although the concentrations of many proteins change with the growth rate, such changes do not necessarily imply that these proteins have been specifically and coordinately up-regulated.
Instead, we show that this change can be the result of suppression of the expression of other proteins and thus does not require further cellular control mechanisms.
Furthermore, we show how the resulting growth rate can be an outcome of the change in the proteome composition suggesting that proteome composition is the cause and growth rate is the result.
}

\section{Introduction}
A fundamental system-level challenge for cell physiology is the achievement of proper function in the face of fluctuating environments.
It has been established for many years that in different environments cells differ in many properties, including their shape, size, growth rate, and macromolecular composition \parencite{Maaloe1969, Schaechter1958, Churchward1982, Pedersen1978a, ingraham1983growth,Bremer1987}, with strong interdependence between these parameters.

Early on it was found that the expression of some genes is coordinated with growth rate.
Classical experiments in bacteria, by researchers from what became known as the Copenhagen school, have shown that ribosome concentration increases in proportion to growth rate\parencite{Schaechter1958}.
The search for mechanisms in \emph{E.coli} that underlie this observation yielded several candidates.
Specifically, coordination between ribosome production and growth rate was attributed both to the pools of purine nucleotides \parencite{Gourse1996,Gaal1997}, and the tRNA pools through the stringent response \parencite{Chatterji2001,Brauer2008a}.
The logic behind this observed increase is that, given that translation rates and ribosome occupancy ratios remain relatively constant across conditions, a larger fraction of ribosomes out of the biomass is needed in order to achieve faster growth.

In the last two decades, with the development of the ability to measure genome-wide expression levels, it was found that coordination of protein concentration and growth rate is not limited to ribosomal genes, but is actually much more wide-spread.
In \emph{E.coli}, a coordination between the expression of catabolic and anabolic genes, and the growth rate, was observed, a process mediated by cAMP \parencite{Saldanha2004}.
In \emph{S.cerevisiae}, it was shown that a surprisingly large fraction of the genome changes its expression levels in response to environmental conditions in a manner strongly correlated with growth rate \parencite{Keren2013a,Gasch2000,Castrillo2007, Zaslaver2009a, Berthoumieux2013, Gerosa2013}.
These studies and others raised many fundamental questions regarding the basic nature of gene regulation that are still open.
For example, what is the degree of interconnection between gene expression and growth rate? What are the mechanisms underlying this connection? Is it mostly due to specific mechanisms, each affecting a distinct group of genes (such as the mechanisms detailed above) or is it a more global phenomenon shared across most genes in the genome?

Several studies examining the interplay between global and specific modes of regulation tried to answer these qeustions, suggesting that global factors play a major role in determining the expression levels of genes \parencite{Gasch2000, Klumpp2009a,Scott2010}.
In \emph{E.coli}, this was mechanistically attributed to changes in the pool of RNA polymerase core and sigma factors \parencite{Klumpp2008}.
In \emph{S.cerevisiae}, it was suggested that differences in histone modifications around the replication origins \parencite{regenberg2006} or translation rates \parencite{Gasch2000} across conditions may underlie the same phenomenon.
Important advancements in understanding this process in \emph{E.coli} were achieved by coupling measurements of fluorescent reporters to a model of expression built upon the empirical scaling of different cell parameters (such as gene dosage, transcription rate and cell size) with growth rate [refs].
These studies suggest that the expression of all genes change with growth rate, with different architectures of regulatory networks yielding differences in the direction and magnitude of these changes. 

Despite these advancements, many gaps remain in our understanding of the connection between gene expression and growth rate.
Importantly, whereas many of these studies depict gene expression as a function of growth rate, other studies suggest that the changes in expression temporally precede the changes in growth rate\parencite{levy2007}.
In addition, mechanistic insight and models for organisms other than \emph{E.coli} are still missing.
As such, a need remains for a quantitative model relating gene expression and growth rate, which can provide a baseline for the behavior of endogenous genes in conditions in which they are not differentially regulated.
Such a model will provide a basis on top of which different regulatory aspects can be added.

In this work we analyzed two recently published proteomic data sets of \emph{E.coli} under different growth conditions.
Based on the analysis we present a baseline, organism-independent, model that quantifies the relationship between gene regulation, protein abundance and growth rate.
Our model suggests that positive correlation with growth rate is a system-emerging property that is the result of passive redistribution of resources.
As a result, such an observed correlation does not require specific regulation of the positively correlated proteins and hence, no mechanism needs to exist in order to explain this phenomena.

\section{Results}
\subsection{Analysis of proteomic data}
Having two high-quality data sets at hand, we looked for global trends in the data and specifically trends involving the growth rate under the different conditions analysed.
We measured the pearson correlation of every protein in the two datasets analyzed (denoted by H for Heinmann's data and V for Valgepea's data) against the growth rate (Figure \ref{fig:growthcorr}).
The two data sets show that a large fraction ($x\% $ in H and $y\% $ in V) are significantly positively correlated with the growth rate.
In the H data set, the correlation with growth rate is more significant among annotated proteins than among un-annotated proteins ($x\% $ of annotated proteins have a correlation in the range of $(a,b)$ whereas only $y\% $ of the un-annotated proteins have a correlation within this range).
In both datasets, the proteins that have a high correlation with the growth rate are involved in different cellular functions and span different functional groups (reftable/SI breakdown by function).
We believe that the lower correlation and higher variability found in the H data set is a byproduct of the fact that it contains much more conditions but spans roughly the same range of growth rates.
Furthermore, it maps conditions involving different carbon sources as opposed to the V data set that uses the same carbon source on all measurements.

\begin{figure}[h]
\centering
\includegraphics{GrowthRateCorrelation.pdf}
\caption{
A significant fraction of the proteins have positive correlation with the growth rate.
These proteins span all the functional groups.
Proteins with unknown function show less correlation with growth rate (as well as proteins with low levels of expression, data not shown).
}
\label{fig:growthcorr}
\end{figure}

Next we checked how significant was the response of the proteins that had a significant positive correlation with the growth rate across the conditions measured (hereafter called the High Correlation proteins, abbreviated as HC-proteins).
To that end, we summed up the concentrations of all of the HC-proteins across the conditions measured and compared their total concentration to the growth rate (Figure \ref{fig:globalgrcorr}).
Both datasets presented a significant response ($x\% $ change in total concentration across $y\% $ change of the growth rate) with most of the variability of the total concentration being explaind by the growth rate ($R^2$ of . in H and . in V). 
As before, the H data set presented more noise, probably due to the reasons mentioned above.

\begin{figure}[h]
\centering
\includegraphics{GlobalClusterGRFit.pdf}
\caption{
The global cluster (sum of all proteins with a correlation in the rage $0.4$ to $0.8$ with the growth rate) can be largely explained by the growth rate ($R^2>0.5$).
Both weighted sum and normalized sum are presented.
Weighted sum means the concentrations of all proteins in the group are summed.
Normalized sum means every protein is first normalized to have an average concentration of 1 across the different growth conditions, and then all proteins in the group are summed.
Some of the unexplained variability of the global cluster by the growth rate can indicate errors in growth rate measurements and/or differences in degradation rates across conditions.
}
\label{fig:globalgrcorr}
\end{figure}

Noting that the fact that two proteins have the same correlation with the growth rate does not imply that they are correlated between themselves, neither does it imply that they scale in the same way, we calculated the slope of a linear regression line with the growth rate for every protein in the HC-proteins and plotted the results in Figure \ref{fig:globalfit}.
In order to compensate for the different concentrations of different proteins, the data for every protein was first normalized to have an average concentration of 1 across the different conditions and then the slope of the fit was calculated.
This analysis revealed that most proteins responded in a similar manner to the different conditions.

Lastly, we wanted to check how the response of the HC-proteins relates to the well-studied response of the ribosomal proteins.
To that end, we performed the same analysis of slopes, restricting it to ribosomal proteins alone (Figure \ref{fig:globalfit}).
The analysis revealed that, on average, all of the HC-proteins scale in the same way as the ribosomal proteins do.

\begin{figure}[h]
\centering
\includegraphics{AllProtsVSRibosomalNormalizedSlopes.pdf}
\caption{
    (A) Most proteins have similar, positive, response to the growth rate meaning they maintain their relative proportions across conditions.
    (B) The response of all of the proteins peaks at the same values as the response of the ribosomal proteins.
}
\label{fig:globalfit}
\end{figure}
\subsection{Theoretical model}
In an attempt to explain how such a wide-span positive response occures, we have constructed a model that is able to reproduce these results as the outcome of redistribution of resources of the biosynthesis machinery.
\subsubsection{Distincting between gene specific and global control of expression}
For every protein, the model decouples its resulting concentration as the product of two control mechanisms:
\begin{enumerate}
\item Protein/gene specific controls such as the associated gene's promoter sequence, 5'-UTRs, ribosomal binding site sequence, and factors affecting this specific gene's expression such as transcription factors and riboswitches that react with the relevant gene.
  While some of these controls (such as, for example, the 5'-UTRs) are static, and therefore condition independent, others are dynamic and may differ under different environmental conditions (such as transcription factors state).
\item The global state of the cell, including availability of RNA Polymeraze and cofactors, Ribosomes concentrations etc.
\end{enumerate}
According to the model, every gene, under every environmental condition, is given an 'affinity-for-expression' (or 'intrinsic-strength') score that encapsulates its gene-specific control state under the condition considered (denoted $w_i(c)$, where $c$ refers to the growth condition).
Our model assumes that the biosynthetic resources of the cell (Ribosomes, RNA Polymerases, etc.) are distributed among the genes according to their affinities under the condition at hand.
The resulting protein concentration, under a specific condition, is therefore its specific affinity under the condition, divided by the sum of all the affinities of all of the genes under that same condition.
Thus, if two proteins have the same affinity under some condition, they will have the same concentration under that condition, if protein $A$ has twice the affinity of protein $B$ under the same condition, then $A$'s concentration will be double than $B$'s concentration under that condition, etc.

This relationship can be simply fomulated as follows:
\begin{equation}
  \label{eq:concentration-ratio}
  p_i(c)=\frac{P_i(c)}{P(c)}=\frac{w_i(c)}{\sum_jw_j(c)}
\end{equation}
where $p_i(c)$ denotes the concentration of protein $i$ under condition $c$, $P_i(c)$ denotes the mass of protein $i$ under condition $c$ per cell, and $P(c)$ denotes the total mass of proteins per cell under condition $c$).
\begin{comment}
\begin{figure}[h]
\centering
\includegraphics{CoordinatedRSquareComparison.pdf}
\caption{
  Proteins in the global cluster fit reasonably well to the global cluster itself
}
\label{fig:globalfit}
\end{figure}

\begin{figure}[h]
\centering
\includegraphics{GlobalClusterCorr.pdf}
\caption{
Proteins that have a high correlation (0.4-0.8) with growth rate mostly have even higher correlation to the sum of these proteins (both weighted sum and normalized sum are presented).
Weighted sum means the concentrations of all proteins in the group are summed.
Normalized sum means every protein is first normalized to have an average concentration of 1 across the different growth conditions, and then all proteins in the group are summed.
The higher correlation indicates that their response is coordinated (they scale by the same factor between conditions).
}
\label{globalcorr}
\end{figure}

\begin{figure}[h]
\centering
\includegraphics{GlobalClusterRSquare.pdf}
\caption{
Plotting the $r^2$ distribution shows that a large fraction of the variability of these proteins is captured by the global response.
}
\label{globalrsq}
\end{figure}

\begin{itemize}
\item This happens in multiple organisms and data sets (show yeast, two data sets of coli). Possibly add mRNA measurements ??.
\end{itemize}
\end{comment}

\printbibliography
\end{document}