\documentclass[notitlepage]{article}
\usepackage{color,soul,amsmath,graphicx}
\usepackage[outercaption]{sidecap}
\usepackage[citestyle=numeric,backend=bibtex]{biblatex}
\usepackage[font=small,labelfont=bf]{caption}
\usepackage{verbatim}
\sidecaptionvpos{figure}{c}
\providecommand{\abs}[1]{\lvert#1\rvert}
\providecommand{\norm}[1]{\lVert#1\rVert}
\bibliography{library}

\title{On the quantitative inter-dependence between gene regulation, protein levels and growth rate}
\author{Leeat Keren, Uri Barenholz, Ron Milo}
\date{April 2014}

\begin{document}
\maketitle
\abstract{
  In many microorganisms, the proteome composition changes dramatically as a function of the growth environment.
Furthermore, many of these changes seem to be coordinated with the growth rate.
However, although cellular growth rates, gene expression levels and gene regulation have been at the center of biological research for decades, their quantitative interdependence remains largely unexplained.

Based on two recently published proteome composition datasets for the model microorganism \emph{E.coli} under various growth conditions, we analyzed global trends in the proteome composition as a function of growth rate.
We found that, for a large fraction of the proteins measured, their cellular concentration coordinately increased with the growth rate.
These proteins belong to different functional groups and are involved in different cellular processes.
We developed a simple model that explains how such a widely coordinated increase in concentration of many proteins can be the result of passive redistribution of resources, following active down regulation of a few proteins.
Our model further explains why and how such changes relate to the growth rate under different environmental conditions.

We conclude that, although the concentrations of many proteins change with the growth rate, such changes do not necessarily imply that these proteins have been specifically and coordinately up-regulated, but can also be the default outcome of suppression of the expression of other proteins.
Furthermore, such changes can be quantitatively related to the resulting growth rate.
}

\section{Introduction}
A fundamental system-level challenge for cell physiology is the achievement of proper function in the face of fluctuating environments.
It has been established for many years that in different environments cells differ in many properties, including their shape, size, growth rate, and macromolecular composition \parencite{Maaloe1969, Schaechter1958, Churchward1982, Pedersen1978a, ingraham1983growth,Bremer1987}, with strong interdependence between these parameters.
Early on it was found that the expression of some genes is coordinated with growth rate.
Classical experiments in bacteria, by researchers from what became known as the Copenhagen school, have shown that ribosome concentration increases in proportion to growth rate2.
This increase is primarily the result of changes in protein production rate, with ribosome production increasing as the square of the growth rate1,7.
The search for mechanisms in \emph{E.coli} that underlie this observation yielded several candidates.
Specifically, coordination between ribosome production and growth rate was attributed both to the pools of purine nucleotides 7,8, and the tRNA pools through the stringent response 9,10

\section{Results}

\begin{figure}[h]
\centering
\includegraphics{GrowthRateCorrelation.pdf}
\caption{
A significant fraction of the proteins have positive correlation with the growth rate.
These proteins span all the functional groups.
Proteins with unknown function show less correlation with growth rate (as well as proteins with low levels of expression, data not shown).
}
\label{growth-corr}
\end{figure}

\begin{figure}[h]
\centering
\includegraphics{GlobalClusterGRFit.pdf}
\caption{
The global cluster (sum of all proteins with a correlation in the rage 0.4 to 0.8 with the growth rate) can be largely explained by the growth rate ($R^2>0.5$).
Both weighted sum and normalized sum are presented.
Weighted sum means the concentrations of all proteins in the group are summed.
Normalized sum means every protein is first normalized to have an average concentration of 1 across the different growth conditions, and then all proteins in the group are summed.
Some of the unexplained variability of the global cluster by the growth rate can indicate errors in growth rate measurements and/or differences in degradation rates across conditions.
}
\label{global-grcorr}
\end{figure}

\begin{figure}[h]
\centering
\includegraphics{AllProtsVSRibosomalNormalizedSlopes.pdf}
\caption{
    (A) Most proteins have similar, positive, response to the growth rate meaning they maintain their relative proportions across conditions.
    (B) The response of all of the proteins peaks at the same values as the response of the ribosomal proteins.
}
\label{global-fit}
\end{figure}

\begin{comment}
\begin{figure}[h]
\centering
\includegraphics{CoordinatedRSquareComparison.pdf}
\caption{
  Proteins in the global cluster fit reasonably well to the global cluster itself
}
\label{global-fit}
\end{figure}

\begin{figure}[h]
\centering
\includegraphics{GlobalClusterCorr.pdf}
\caption{
Proteins that have a high correlation (0.4-0.8) with growth rate mostly have even higher correlation to the sum of these proteins (both weighted sum and normalized sum are presented).
Weighted sum means the concentrations of all proteins in the group are summed.
Normalized sum means every protein is first normalized to have an average concentration of 1 across the different growth conditions, and then all proteins in the group are summed.
The higher correlation indicates that their response is coordinated (they scale by the same factor between conditions).
}
\label{global-corr}
\end{figure}

\begin{figure}[h]
\centering
\includegraphics{GlobalClusterRSquare.pdf}
\caption{
Plotting the $r^2$ distribution shows that a large fraction of the variability of these proteins is captured by the global response.
}
\label{global-rsq}
\end{figure}

\begin{itemize}
\item This happens in multiple organisms and datasets (show yeast, two datasets of coli). Possibly add mRNA measurements ??.
\end{itemize}
\end{comment}

\printbibliography
\end{document}