\documentclass[a4paper,landscape,titlepage,17pt]{extarticle}
\usepackage{color,amsmath,graphicx}
\usepackage[citestyle=numeric,backend=bibtex]{biblatex}
\usepackage[labelfont=bf]{caption}
\usepackage[affil-it]{authblk}
\usepackage{fullpage}
\linespread{1}
\pagenumbering{gobble}
\bibliography{library}

\title{\Huge Increased concentration of proteins with growth rate can result from passive resource redistribution}
\author{\Large Uri Barenholz, Leeat Keren, Ron Milo}
\affil{The Weizmann Institute of Science, Rehovot, Israel}
\date{}

\begin{document}
\maketitle
\section*{\LARGE Abstract}
In many microorganisms, the proteome composition changes dramatically as a function of the growth environment.
Furthermore, many of these changes seem to be coordinated with the growth rate rather than the specific environment.
However, although cellular growth rates, gene expression levels and gene regulation have been at the center of biological research for decades, their quantitative interdependence is not yet fully understood.

We analyzed the relationship between growth rate and proteome composition for the model microorganism \emph{E.coli} as reflected in two proteomics data sets spanning various growth conditions.
We found that the cellular concentration of a large fraction of the proteins measured coordinately increases with the growth rate.
This fraction includes proteins spanning different functional groups and proteins that are involved in different cellular processes.
Notably, Ribosomal proteins are only a small fraction of this group of proteins.

We present a simple model that demonstrates how such a widely coordinated increase in the concentration of many proteins can be the result of passive redistribution of resources, due to active regulation of a few proteins.
Our model provides a potential explanation for why and how such changes relate to the growth rate under different environmental conditions.
The model thus suggests that, although the concentrations of many proteins change with the growth rate, such changes could be part of a global effect, not requiring specific cellular control mechanisms.
\clearpage        
\section*{\LARGE Results}
\subsection*{Many proteins are positively correlated with growth rate}
The Pearson correlation between the concentration of every protein and the growth rate in the two data sets (denoted by H for Heinmann's data and V for Valgepea's data) is shown in Figure \ref{fig:growthcorr}.
The two data sets present a large number of proteins ($724$ out of $1981$ measured in H and $296$ out of $2184$ in V) that are significantly positively correlated with the growth rate.
In the H data set, the correlation with growth rate is more significant among annotated proteins than among un-annotated proteins ($435$ out of $988$ of annotated proteins have a correlation in the range of $(0.4,0.8)$ whereas only $289$ out of $993$ of the un-annotated proteins have a correlation within this range).
In both data sets, the proteins that have a high correlation with the growth rate are involved in different cellular functions and span different functional groups.

The lower correlation and higher variability found in the H data set may result from the fact that it contains much more conditions but spans roughly the same range of growth rates.
Furthermore, it includes measurements of the proteome under conditions involving different carbon sources as opposed to the V data set that uses the same carbon source in all measurements.
This is exemplified by restricting the analysis of the H data set only to chemostat conditions (Figure \ref{fig:growthcorrchemo}).


\begin{figure}[h]
\centering
\includegraphics[scale=1.9]{GrowthRateCorrelation.pdf}
\caption{\linespread{0.5}\selectfont{}
A significant fraction of the proteins have positive correlation with the growth rate in the two data sets analyzed.
These proteins span many functional groups.
Proteins with unknown function (present in the Heinmann data set, right panel) show less correlation with growth rate, as well as proteins with low levels of expression (data not shown).
}
\label{fig:growthcorr}
\end{figure}

\clearpage        
\subsection*{Proteins that are positively correlated with growth rate share a similar, significant response}
We checked how large is the response of the proteins that have a significant positive correlation with the growth rate across the conditions measured (referred to as HC, for High Correlation, proteins).
To that end, we summed up the concentrations of all of the HC proteins across the conditions measured and compared their total concentration to the growth rate (Figure \ref{fig:globalgrcorr}).
Both data sets presented a significant response ($\approx 2$ fold change in total concentration across $\approx 5$ fold change of the growth rate) with most of the variability of the total concentration being explained by the growth rate ($R^2$ of $0.75$ in H and $0.99$ in V). 
As before, the H data set presented more noise, due to the reasons mentioned above (see Figure \ref{fig:growthcorrchemo} for analysis of the chemostat conditions of the H data set).

Noting that the fact that two proteins have the same correlation with the growth rate does not imply that they are correlated between themselves, neither does it imply that they scale in the same way, we calculated the slope of a linear regression line with the growth rate for every protein in the HC proteins and plotted the results in Figure \ref{fig:globalfit}.
In order to compensate for the different concentrations of different proteins, the data for every protein was first normalized to have an average concentration of $1$ across the different conditions and then the slope of the fit was calculated.
The analysis reveals that most proteins respond in a similar manner across the different conditions.

\subsection*{Ribosomal proteins share the same response to growth rate as other proteins}
We wanted to check how the response of the HC proteins relates to the well-studied response of the ribosomal proteins.
To that end, we performed the same analysis of slopes, restricting it to ribosomal proteins alone (Figure \ref{fig:globalfit}).
The analysis shows that, on average, HC proteins scale in the same way as ribosomal proteins do.
\clearpage        

\begin{figure}[h]
\centering
\includegraphics[scale=1.7]{GlobalClusterGRFit.pdf}
\caption{\linespread{0.5}\selectfont{}
The sum of the concentrations of HC proteins in each of the data sets (with linear regression lines) is shown.
HC proteins form a large fraction out of the proteome at higher growth rates ($>40\%$ for H and $>50\%$ for V).
The change in the concentration of the HC proteins is about 2-fold while the growth rate changes by about 5-fold.
Some of the unexplained variability of the HC proteins in the H data set can indicate errors in growth rate measurements and/or differences in degradation rates across conditions.
(HC proteins are defined as those with a Pearson correlation in the range $(0.4,0.8)$ and $(0.8,1)$ for the H and V data sets, respectively)
}

\label{fig:globalgrcorr}
\end{figure}
\clearpage        
\begin{figure}[h]
\centering
\includegraphics[scale=1.9]{AllProtsVSRibosomalNormalizedSlopes.pdf}
\caption{\linespread{0.5}\selectfont{}
    A histogram of the normalized slopes of the trend lines for every protein in the HC proteins and for all the ribosomal proteins is shown for the two data sets analyzed.
    Left panel - Valgepea, right panel - Heinmann.
    The HC proteins all share a relatively similar response, meaning they maintain their relative ratios.
    The ribosomal proteins respond in a similar manner to the HC proteins .
}
\label{fig:globalfit}
\end{figure}
\clearpage        

\begin{figure}[h]
\centering
\includegraphics[scale=1.9]{HeinmannChemostatGr.pdf}
\caption{\linespread{0.5}\selectfont{}
  Restricting the analysis of the Heinmann data set to chemostat conditions yields similar results to those obtained for the Valgepea data set.
}
\label{fig:growthcorrchemo}
\end{figure}
\clearpage        

\subsection*{Theoretical model}
In an attempt to explain how such a wide-span positive response occurs, we have constructed a model that is able to reproduce these results as the outcome of redistribution of resources of the bio-synthesis machinery.
Briefly, the model assumes that favorable growth conditions allow the cell to down-regulate some proteins that are needed in harsher conditions, thus reducing the amount of proteins that need to be produced for the cell to proliferate.
As a consequence, the fraction of each of the rest of the proteins out of the proteome increases, assuming they are being expressed in the same relative ratios.
The growth rate therefore increases as well, due to the fact that the ratio of bio-synthetic machinery to the rest of the proteome increases.

According to the model, for non-differentially regulated proteins, their concentration should drop to 0 when the growth rate is 0.
Interestingly, protein degradation affects the expected concentration of non-differentially regulated proteins at 0 growth rate.

Assuming that protein degradation acts on all proteins in the same way, and that it is invariant in the growth rate, the effect of protein degradation can be understood as follows: at any time, some fraction of the entire proteome is degraded.
Therefore, the \emph{observed} growth rate, $\mu$, is, in fact, equivalent to the amount of proteins produced minus the amount of proteins degraded.
To illustrate, if the measured growth rate is 0, the implication is not that no proteins are produced, but rather that proteins are produced at exactly the same rate as they are degraded.

\printbibliography
\end{document}